gloo/docs/gloo_routing/virtual_services/rate_limiting/envoy/_index.md

```diff
@@ -240,12 +240,14 @@ You can run the same command for a *route* as well (`glooctl edit route ...`). W
 
 ### Test
 
-You can use `glooctl proxy url` to get Gloo proxy's external endpoint. Run `curl --head $URL` a few times and eventually the curl response shows it is being rate limited:
+Run the following a few times:
 
 ```bash
-curl --head $URL
+curl --head $(glooctl proxy url)
```

Comment:
I remember us already talking about this. It think `glooctl proxy url` will currently error, as no service called `gateway-proxy` exists. This might not be future-proof, but I would update this:

```suggestion
curl --head $(glooctl proxy url --name gateway-proxy-v2)
``

We can full-text-search and revert back to the previous version once [this issue](https://github.com/solo-io/gloo/issues/1123) has been solved.